tests/cucumber/fixtures/accounts_fixtures.js

```diff
@@ -0,0 +1,19 @@
+(function () {
+
+  'use strict';
+
+  Meteor.methods({
+    'reset' : function() {
+      Meteor.users.remove({});
+      UserProfiles.remove({});
+    },
+
+    'createProfile' : function(field, value) {
+      var attributes = {};
+      attributes[field] = value;
+      attributes['_id'] = 'fakeid';
```

Comment:
I can't think of a circumstance where you'd need to specify an ID rather than just know what the ID is.  That said, I like the idea of this method taking an argument and the step implementation calling this method with a hardcoded value.